src/Fixer/Basic/PsrAutoloadingFixer.php

```diff
@@ -137,6 +137,10 @@ protected function createConfigurationDefinition()
      */
     protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
+        if (null !== $this->configuration['dir'] && 0 !== strpos($file->getRealPath(), realpath($this->configuration['dir']))) {
```

Comment:
without any test added, it's hard to see what you are fixing.
please, add a test

Reply:
Done.